client/src/styles/legacy/_badges.scss

```diff
@@ -0,0 +1,16 @@
+.badge {
+  display: inline-block;
+  background-color: $color-notice;
+  border-color: transparent;
+  color: $body-color-dark;
+  // Badges with no text aren't displayed
+  &:empty {
+    display: none;
+  }
+}
```

Comment:
This already gets inherited from bootstrap, but it surprised me at first. Adding it here so it's explicitly intended behaviour.